src/sql/workbench/contrib/query/browser/query.contribution.ts

```diff
@@ -147,12 +147,13 @@ actionRegistry.registerWorkbenchAction(
 
 actionRegistry.registerWorkbenchAction(
 	SyncActionDescriptor.create(
-		RunCurrentQueryWithActualPlanKeyboardAction,
-		RunCurrentQueryWithActualPlanKeyboardAction.ID,
-		RunCurrentQueryWithActualPlanKeyboardAction.LABEL,
+		ToggleActualPlanKeyboardAction,
+		ToggleActualPlanKeyboardAction.ID,
+		ToggleActualPlanKeyboardAction.LABEL,
 		{ primary: KeyMod.CtrlCmd | KeyCode.KEY_M }
 	),
-	RunCurrentQueryWithActualPlanKeyboardAction.LABEL
+	ToggleActualPlanKeyboardAction.LABEL,
+	localize('ExecutionPlan', 'Execution Plan')
```

Comment:
Typically if you find yourself copying the same constant (especially localized constants) it's better to try and consolidate them into a single instance that everything references. This not only ensures they all share the same name (we don't miss updating any if we change it in the future) but also saves the loc team from having to localize the same string multiple times. 

VS Code does this for general categories it looks like for example :  https://github.com/Microsoft/azuredatastudio/blob/2bc6a0cd012de69504061a876070cfa43533fedc/src/vs/workbench/common/actions.ts#L128